backend/src/main/java/de/christianporsch/backend/model/Price.java

```diff
@@ -0,0 +1,16 @@
+package de.christianporsch.backend.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class Price {
+
+    private double usd;
+    private double eur;
+    private double tix;
```

Comment:
In Java you should use BigDecimal for prices.
😉😉
[monetary-calculation](https://dzone.com/articles/never-use-float-and-double-for-monetary-calculatio)